packages/data-context/src/actions/MigrationActions.ts

```diff
@@ -34,11 +34,11 @@ import {
 } from '../sources/migration'
 import { makeCoreData } from '../data'
 import { LegacyPluginsIpc } from '../data/LegacyPluginsIpc'
-import { hasTypeScriptInstalled } from '../util'
+import { hasTypeScriptInstalled, toPosix } from '../util'
 
 const debug = debugLib('cypress:data-context:MigrationActions')
 
-const tsNode = require.resolve('@packages/server/lib/plugins/child/register_ts_node')
+const tsNode = toPosix(require.resolve('@packages/server/lib/plugins/child/register_ts_node'))
```

Comment:
Is this helper necessary? Can we use path.joint out of the box?
```suggestion
const tsNode = require.resolve(path.join('@packages', 'server', 'lib','plugins','child', 'register_ts_node'))
```

Reply:
@emilyrohrbough I believe the intent here is to properly escape the actual resolved path that's returned from `require.resolve`, which `path.join` wouldn't help with

Reply:
Yes, the helper is the fix. We need to posix-ify the path, even on windows.